_drafts/unit-testing-with-mockito.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title:  "Unit Testing with Mockito for Android"
+author: johnleeroy
+description: Learn how to unit test your Android code with JUnit4 and Mockito.
+categories: 
+  - android
+  - java 
+  - unit testing
+  - tdd
+  - code coverage
+  - tech
+---
+Every developer knows that testing software can take a lot of time.  Being the brilliant industrious individuals that we are, engineers invented
+multiple ways to test software.  Unit testing is just one part among others such as functional testing, integration testing, and acceptance testing.
+In this blog, we will go over how to write unit tests for Android using the latest tools and libraries.  In this example, I am leverage JUnit4 and Mockito.  
+
+# What is Unit Testing?
```

Comment:
Let's use at least `##` for subtitles. So we can leave `<h1>` tags for the blog title.